templates/repo/settings/protected_branch.tmpl

```diff
@@ -104,6 +104,38 @@
 					{{end}}
 					</div>
 
+					<div class="field">
+						<div class="ui checkbox">
+							<input class="enable-statuscheck" name="enable_status_check" type="checkbox" data-target="#statuscheck_contexts_box" {{if .Branch.EnableStatusCheck}}checked{{end}}>
+							<label>{{.i18n.Tr "repo.settings.protect_check_status_contexts"}}</label>
```

Comment:
Since this is a headline, it should be something like `Require passing checks before merging` and not `Status checks found in the last week for this repository`

Reply:
As above.